### PR TITLE
Display lock icon next to a sealed case in order search results UI

### PR DESCRIPTION
**Pull request by mrinsin**
image

### CODE REVIEW

File: web-client/src/styles/icons.scss [hunk 1]
```diff
@@ -37,6 +37,13 @@ button:not(.ustc-button--unstyled) .fa-file-pdf {
   color: color($theme-color-warning);
 }
 
+.iconSealed {
+  margin-top: 4px;
+  margin-right: -22px;
+  color: #b51d09;
```
Comment (rachaelparris) [inline web-client/src/styles/icons.scss]: For this color, you'll need to use the format `color: color($theme-color-warning);`. I'm not sure what color this is, but the USWDS docs would be a good place to look. I can help you with that if you haven't done that before.

File: web-client/src/views/AdvancedSearch/OrderSearchResults.jsx [hunk 2]
```diff
@@ -42,7 +43,17 @@ export const OrderSearchResults = connect(
                 {advancedOrderSearchHelper.formattedSearchResults.map(
                   (result, idx) => (
                     <tr className="search-result" key={idx}>
-                      <td aria-hidden="true">{idx + 1}</td>
+                      <td aria-hidden="true">
+                        {idx + 1}
+                        {result.isSealed && (
```
Comment (rachaelparris) [inline web-client/src/views/AdvancedSearch/OrderSearchResults.jsx:51]: I think we typically create a separate column for icons for accessibility - similar to how to the docket record table is set up.

File: web-client/src/styles/icons.scss (hunk 1 above)
Comment (mrinsin) [reply web-client/src/styles/icons.scss]: Hey- I would love some help on that, thank you!  I was trying to look for the file where the theme color variables are defined but was unsuccessful!

Comment (rachaelparris) [reply web-client/src/styles/icons.scss]: It's a bit confusing - most of them are defined here: https://designsystem.digital.gov/design-tokens/color/theme-tokens/

and here: https://designsystem.digital.gov/design-tokens/color/state-tokens/

Comment (mrinsin) [reply web-client/src/styles/icons.scss]: Thank you! I must have missed your response but I was still able to find it!

**Review by rachaelparris — APPROVED** [review]

**Pull request merged**